README.md

```diff
@@ -1,22 +1,34 @@
-# Repository template
-A template enabled repository, including all necessary files to open source.
-(create an issue with the following content if you want to track the repo configuration)
-
-## Repository settings and configuration
-- [ ]  Repository info
-    - [ ]  Add repo description
-    - [ ]  Update website to https://onflow.org
-    - [ ]  Add relevant repository topics (i.e. `blockchain` `onflow`, etc)
-    - [ ]  Check issue labels on `.github/labels.yml` and do any commit to main to get them synced
-- [ ]  Define merge workflow (create new branch protection rule)
-    - [ ]  `main` branch rule:
-        - [ ]  **Require pull request reviews before merging (2 approving reviews)**
-            - [ ]  **Require review from Code Owners**
-        - [ ]  **Require status checks to pass before merging**
-            - [ ]  **Require branches to be up to date before merging**
-        - [ ]  **Require linear history**
-        - [ ]   **Restrict who can push to matching branches**
-            - [ ]  Choose `onflow/flow` team
-- [ ]  Add necessary team members, adjust access levels
-    - [ ]  `onflow/flow-admin` ⇒ Admin access
-    - [ ]  `onflow/flow-engineering ` ⇒ Write access
\ No newline at end of file
+## Flow Solidity Utilities
+
+This repository contains a collection of Solidity contracts uniquely useful to building on EVM on Flow. You'll find
+libraries, interface and abstract contracts, as well as contract implementations you may find useful in your own
+projects.
+
+### Contracts
+
+* WFLOW - the equivalent of WETH on Ethereum, a wrapped version of FLOW token.
```

Comment:
might want to add testnet and mainnet deployment addresses to this table

Reply:
Good call, added in [0e0bead](https://github.com/onflow/flow-sol-utils/pull/3/commits/0e0beadfb89afbd3ebc58edb1ecb6ea61db5f676)